src/components/titles/index.js

```diff
@@ -14,6 +14,8 @@ var interactConstants = require('../../constants/interactions');
 
 var OPPOSITE_SIDE = require('../../constants/alignment').OPPOSITE_SIDE;
 var numStripRE = / [XY][0-9]* /;
+var MATHJAX_PADDING_MULTIPLIER = 0.85;
+var EXTRA_SPACING_BETWEEN_TITLE_AND_SUBTITLE = 0;
```

Comment:
Although I kind of like the current positioning of subtitle which is compact;
I was wondering the correct value for this constant might be 1.3
https://github.com/plotly/plotly.js/blob/8b1f5bd35c1b7cd388025b01a4d6a1abdb3ff9ae/src/constants/alignment.js#L34

This is why: 
If I call 
```js
Plotly.newPlot(gd, [{y: [1, 2]}], {title: {text: 'title', subtitle: {text: 'subtittle'}}});
```
the subtitle render closer compared to when I call
```js
Plotly.newPlot(gd, [{y: [1, 2]}], {title: {text: 'title<br><sub>subtittle</sub>'}}); 
```

So perhaps you could try using `(LINE_SPACING - 1) * title.font.size` in your calculations?

Reply:
@archmoj Do you think the subtitle looks better with additional spacing?

I'd prefer not to change the spacing _just_ to match the appearance of `<sub>`, since that spacing is arbitrary.

I prefer the more compact look.